client/my-sites/site-settings/disconnect-site/why/disconnect-reasons-menu.jsx

```diff
@@ -0,0 +1,99 @@
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Card from 'components/card';
+import SelectDropdown from 'components/select-dropdown';
+import PaginationFlow from './pagination-flow';
+import { isJetpackSite } from 'state/sites/selectors';
+import { getSelectedSiteId, getSelectedSiteSlug } from 'state/ui/selectors';
+import DisconnectReasonsSuggestion from './disconnect-reasons-suggestion';
+
+class DisconnectReasonsMenu extends Component {
+	constructor( props ) {
+		super( props );
+		const initialState = {
+			childSelected: 'missingFeature',
+			compactButtons: false,
+			renderFull: false
+		};
+
+		this.state = initialState;
+		this._logReason = this._logReason.bind( this );
+	}
+
+	displayFull = ( option ) => {
+		return (
+			<DisconnectReasonsSuggestion
+				optionSelected={ option }
+				compactButtons={ this.state.compactButtons } />
+		);
+	}
+
+	_logReason = ( option ) => {
+		this.setState( {
+			childSelected: option.value,
+			renderFull: true
+		} );
+	}
+
+	render() {
+		const {
+			translate,
+			siteSlug
+				} = this.props;
+		const {
+			childSelected,
+			compactButtons,
+			renderFull
+		} = this.state;
+
+		const textShareWhy = translate( 'Would you mind sharing why you want to' +
```

Comment:
This one will be a problem for translators to understand, and possibly to translate in some languages, because you've split it to three parts with two translatable strings. Each string is individually translated, and those two translatable strings are not necessarily explainable enough without the context of each other.

So, I'd recommend you combining this one in a single `translate()` call.